docs/user/alerting/alerting-troubleshooting.asciidoc

```diff
@@ -1,263 +1,199 @@
-
-| If another {kib} instance with a different encryption key connects to the cluster.
-| The other {kib} instance might be trying to run the rule using a different encryption key than what the rule was created with. Ensure the encryption keys among all the {kib} instances are the same, and setting <<xpack-encryptedSavedObjects-keyRotation-decryptionOnlyKeys, decryption only keys>> for previously used encryption keys.
-
-| If other scenarios don't apply.
-| Generate a new API key for the rule by disabling then enabling the rule.
-
-|===
+[source, txt]
+--------------------------------------------------
+server    log   [11:39:40.389] [error][alerting][alerting][plugins][plugins] Executing Alert "5b6237b0-c6f6-11eb-b0ff-a1a0cbcf29b6" has resulted in Error: Saved object [action/fdbc8610-c6f5-11eb-b0ff-a1a0cbcf29b6] not found
+--------------------------------------------------
+Some of the resources, such as saved objects and API keys, may no longer be available or valid, yielding error messages about those missing resources.
 
 [float]
-[[rules-small-check-interval-run-late]]
-=== Rules with small check intervals run late
+[[alerting-kibana-version]]
+=== Use the debugging tools
+The following debugging tools are available:
 
-*Problem*:
+* {kib} versions 7.10 and above
+have a <<testing-connectors,Test connector>> UI.
 
-Rules with a small check interval, such as every two seconds, run later than scheduled.
-
-*Resolution*:
-
-Rules run as background tasks at a cadence defined by their *check interval*.
-When a Rule *check interval* is smaller than the Task Manager <<task-manager-settings,`poll_interval`>> the rule will run late.
-
-Either tweak the <<task-manager-settings,{kib} Task Manager settings>> or increase the *check interval* of the rules in question.
-
-For more details, see <<task-manager-health-scheduled-tasks-small-schedule-interval-run-late>>.
+* {kib} versions 7.11 and above
+include improved Webhook error messages,
+better overall debug logging for actions and connectors,
+and Task Manager <<task-manager-diagnosing-root-cause,diagnostics endpoints>>.
 
 [float]
-[[scheduled-rules-run-late]]
-=== Rules run late
-
-*Problem*:
+[[alerting-managment-detail]]
+=== Using rules and connectors list for the current state and finding issues
+*Rules and Connectors* in *Stack Management* lists the rules and connectors available in the space you’re currently in. When you click a rule name, you are navigated to the <<rule-details,details page>> for the rule, where you can see currently active alerts. 
+The start date on this page indicates when a rule is triggered, and for what alerts. In addition, the duration of the condition indicates how long the instance is active.
 
-Scheduled rules run at an inconsistent cadence, often running late.
-
-Actions run long after the status of a rule changes, sending a notification of the change too late.
-
-*Solution*:
+[float]
+[[alerting-index-threshold-chart]]
+=== Preview the index threshold rule chart
```

Comment:
This section talks about the index threshold preview chart but the screenshot is of the rule details page. There is no end date in the rule details page.